packages/terra-site/CHANGELOG.md

```diff
@@ -3,6 +3,10 @@ Changelog
 
 Unreleased
 ----------
+1.22.1 - (January 20, 2018)
```

Comment:
You can leave the date off for now. We update that line when we release the package

Reply:
Good catch. Removed it in 22def6f.